packages/swap/prisma/migrations/20240711155734_migrate_bitcoin_transaction_ref/migration.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE "public"."Broadcast" SET "transactionRef" = SUBSTRING("transactionRef", 3) WHERE "chain" = 'Bitcoin' AND "transactionRef" IS NOT NULL;
```

Comment:
is 3 correct here? shouldn't this be substring( ..., 2) ?

Reply:
surprisingly yes - if you pass 1 the first character will still be included 🙈